<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InSaveEnvDlgClass</class>
 <widget class="QDialog" name="InSaveEnvDlgClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InSaveEnvDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lb_OutEnv">
           <property name="text">
            <string>OutEnv</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="tv_OutEnv"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_CopyToLeft">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_CopyToRight">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lb_InEnv">
           <property name="text">
            <string>InEnv</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="tv_InEnv"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>OutSelected</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lb_OutSelected">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>InSelected</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lb_InSelected">
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="insaveenvdlg.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tv_InEnv</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>InSaveEnvDlgClass</receiver>
   <slot>InEnvSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tv_OutEnv</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>InSaveEnvDlgClass</receiver>
   <slot>OutEnvSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_CopyToLeft</sender>
   <signal>clicked()</signal>
   <receiver>InSaveEnvDlgClass</receiver>
   <slot>CopyToOutEnv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_CopyToRight</sender>
   <signal>clicked()</signal>
   <receiver>InSaveEnvDlgClass</receiver>
   <slot>CopyToInEnv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>InEnvSelected(QModelIndex)</slot>
  <slot>OutEnvSelected(QModelIndex)</slot>
  <slot>CopyToOutEnv()</slot>
  <slot>CopyToInEnv()</slot>
 </slots>
</ui>
